Releases.md

```diff
@@ -27,6 +28,43 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 | [1.1.0](#directml-110)   | [DML_FEATURE_LEVEL_2_0](https://learn.microsoft.com/windows/ai/directml/dml-feature-level-history#dml_feature_level_2_0) | Windows 10 May 2020 Update, Version 2004 (Build 10.0.19041, 20H1) | -                                                                                           |
 | [1.0.0](#directml-100)   | [DML_FEATURE_LEVEL_1_0](https://learn.microsoft.com/windows/ai/directml/dml-feature-level-history#dml_feature_level_1_0) | Windows 10 May 2019 Update, Version 1903 (Build 10.0.18362; 19H1) | -                                                                                           |
 
+# DirectML 1.15.0
+
+- Add support for WebNN dev preview models
```

Comment:
DirectML already supported these models the past year, and so that's not new. What's significant is *NPU* support. Mention it?

```suggestion
- Added NPU support for WebNN dev preview models
```
or
```suggestion
- Improved NPU support for WebNN dev preview models
```